iocore/hostdb/HostDB.cc

```diff
@@ -2224,13 +2227,14 @@ put_hostinfo_ClusterFunction(ClusterHandler *ch, void *data, int /* len ATS_UNUS
 int
 HostDBContinuation::backgroundEvent(int /* event ATS_UNUSED */, Event * /* e ATS_UNUSED */)
 {
-  ++hostdb_current_interval;
+  // No nothing if hosts file checking is not enabled.
+  if (hostdb_hostfile_check_interval == 0) {
+    return EVENT_CONT;
```

Comment:
Shouldn't this be `EVENT_DONE`? Because it's done, there is no continuing operation.